src/bots/modchat/tests/messageHandler.test.js

```diff
@@ -0,0 +1,54 @@
+import client from '../../../client';
+import handlePrivateMessage from '../messageHandler';
+// import * as utils from '../../../utils/index';
+
+describe('handlePrivateMessage', () => {
+  test('messages when user provides no arguments', async () => {
+    client.message.content = '++mc';
+
+    await handlePrivateMessage();
+
+    expect(client.message.channel.send).toHaveBeenCalledWith(
+      'Direct message bot with ++mc -a <message> to privately message mods.\nRemove -a flag to include discord name (otherwise anonymous).'
+    );
+  });
+
+  test('messages when user does not use DMChannel', async () => {
+    client.message.content = '++mc hello world';
+
+    await handlePrivateMessage();
+
+    expect(client.message.channel.send).toHaveBeenCalledWith(
+      'Please send this through direct message to bot'
+    );
+  });
+
+  /*test.skip('sends success message when everything is sent', async () => {
```

Comment:
should this be removed?

Reply:
I want to keep this because it is a valid test minus not able to cast to DMChannel

Reply:
ok, can we actually remove the skip rule from lint then? would prefer being able to use skip instead of commenting large blocks of code

Reply:
yeah I thought there was some reason you wanted it in